container-aws-yaml/Pulumi.yaml

```diff
@@ -0,0 +1,72 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: yaml
+template:
+  description: A containerized service on AWS Fargate
+  config:
+    aws:region:
+      description: The AWS region to deploy into
+      default: us-west-2
+    containerPort:
+      description: The port to expose on the container
+      default: 80
+    cpu:
+      description: The amount of CPU to allocate for the container
+      default: 512
+    memory:
+      description: The amount of memory to allocate for the container
+      default: 128
+
+configuration:
+  containerPort:
+    type: Number
+    default: 80
+  cpu:
+    type: Number
+    default: 512
+  memory:
+    type: Number
+    default: 128
+
+resources:
+  # An ECS cluster to deploy into
+  cluster:
+    type: aws:ecs:Cluster
+  # An ALB to serve the container endpoint to the internet
+  loadbalancer:
+    type: awsx:lb:ApplicationLoadBalancer
+    options:
+      version: 1.0.0-beta.10
+  # An ECR repository to store our application's container image
+  repo:
+    type: awsx:ecr:Repository
+    options:
+      version: 1.0.0-beta.10
+  # Build and publish our application's container image from ./app to the ECR repository
+  image:
+    type: awsx:ecr:Image
+    properties:
+      repositoryUrl: ${repo.url}
+      path: ./app
+    options:
+      version: 1.0.0-beta.10
+  # Deploy an ECS Service on Fargate to host the application container
+  service:
+    type: awsx:ecs:FargateService
+    properties:
+      cluster: ${cluster.arn}
+      taskDefinitionArgs:
+        container:
+          image: ${image.imageUri}
+          cpu: ${cpu}
+          memory: ${memory}
+          essential: true
+          portMappings:
+          - containerPort: ${containerPort}
+            targetGroup: ${loadbalancer.defaultTargetGroup}
+    options:
+      version: 1.0.0-beta.10
+
+outputs:
+  # The URL at which the container's HTTP endpoint will be available
+  url: http://${loadbalancer.loadBalancer.dnsName}
```

Comment:
This line seems to be causing issues for me:

![image](https://user-images.githubusercontent.com/274700/191128800-60b8298a-7943-4443-bfa0-862a8da793ef.png)

It _looks_ right, though, compared to others that do work, so I'm not sure why this one doesn't.

Reply:
Actually something else must be going on here. Removing that line seems to fail in the same way. 🤔